.github/workflows/snyk_push.yml

```diff
@@ -0,0 +1,27 @@
+---
+name: Snyk Clojure Merge
+
+on: push
+
+jobs:
+ snyk_clojure:
+   runs-on: ubuntu-latest
+   steps:
+    - name: checkout the current branch head
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 1
+        persist-credentials: false
+    - name: Run Clojure Snyk Scan
+      id: scan
+      uses: puppetlabs/security-snyk-clojure-action@v1
+      with:
+        snykToken: ${{ secrets.SNYK_PE_TOKEN }}
+        rproxyKey: ${{ secrets.SEC_PUBLIC_RPROXY_KEY }}
+        rproxyUser: ${{ secrets.SEC_PUBLIC_RPROXY_USER }}
+        snykOrg: 'puppet-enterprise'
+        snykProject: 'puppetserver'
+        snykPolicy: '.snyk'
```

Comment:
Should the PR workflow have this policy too? I noticed some of Steve's PRs had this commented out, so wasn't sure what that was about.

Reply:
Jon's have left it in, even with no file present. So I copied that.